### ISSUE TITLE
new hook failed for AWS when there is no feature selected

### ISSUE THREAD

**Issue by xiujiao** [genesis-community/bosh-genesis-kit#93]
When run `genesis new` in AWS, the following error occured
```
What Blobstore do you want to use?
  1) Create a local blobstore on the BOSH director VM (default)
  2) Use an existing AWS S3 Blobstore

Select choice > Create a local blobstore on the BOSH director VM
./hooks/new: line 495: features[@]: unbound variable
[ERROR] Could not create new env snw-drgao-aws-test (in ~/work/sw/deployments/bosh): 'new' hook exited 1
```

**Comment by bodymindarts** [comment]
I just ran into the same issue - is there a work around?

**Comment by xiujiao** [comment]
> ```
> Select choice > Create a local blobstore on the BOSH director VM
> ./hooks/new: line 307: features[@]: unbound variable
> [ERROR] Could not create new env proto-aws (in ~/ops/bosh-deployments): 'new' hook exited 1
> ```
> 
> I just ran into the same issue - is there a work around?

I submitted a PR 94 to fix this, but didn't get merged 

@bodymindarts @dennisjbell

**Comment by dennisjbell** [comment]
This was resolved by v2.0.0
